ros2/ubuntu/overlay.repos

```diff
@@ -0,0 +1,9 @@
+repositories:
+  rtt_ros2_integration:
+    type: git
+    url: https://github.com/orocos/rtt_ros2_integration.git
+    version: master
```

Comment:
The current `master` has a bug in the `CMakeLists.txt` of the `tests` subfolder. So this requires to attend first the fix that I submitted [this PR](https://github.com/orocos/rtt_ros2_integration/pull/20).

Reply:
I merged https://github.com/orocos/rtt_ros2_integration/pull/20 after having applied and tested the alternative solution as discussed today.